coordinator/persistence-cassandra-4.1/src/main/java/io/stargate/db/cassandra/impl/StargateSystemKeyspace.java

```diff
@@ -0,0 +1,126 @@
+package io.stargate.db.cassandra.impl;
+
+import com.google.common.collect.ImmutableList;
+import java.nio.ByteBuffer;
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import org.apache.cassandra.config.DatabaseDescriptor;
+import org.apache.cassandra.cql3.CQLStatement;
+import org.apache.cassandra.cql3.QueryProcessor;
+import org.apache.cassandra.cql3.statements.SelectStatement;
+import org.apache.cassandra.db.SystemKeyspace;
+import org.apache.cassandra.db.virtual.*;
+import org.apache.cassandra.dht.IPartitioner;
+import org.apache.cassandra.dht.Token.TokenFactory;
+import org.apache.cassandra.locator.InetAddressAndPort;
+import org.apache.cassandra.schema.*;
+import org.apache.cassandra.transport.ProtocolVersion;
+import org.apache.cassandra.utils.FBUtilities;
+import org.apache.cassandra.utils.MurmurHash;
+
+public class StargateSystemKeyspace extends VirtualKeyspace {
+  public static final String SYSTEM_KEYSPACE_NAME = "stargate_system";
+  public static final String LOCAL_TABLE_NAME = "local";
+  public static final String LEGACY_PEERS_TABLE_NAME = "peers";
+  public static final String PEERS_V2_TABLE_NAME = "peers_v2";
+
+  public static final UUID SCHEMA_VERSION = UUID.fromString("17846767-28a1-4acd-a967-f609ff1375f1");
```

Comment:
As with anything else, this is copied from an earlier place but... does anyone know where did this come from?
If so, would be good comment addition.